### DISCUSSION TITLE
You've saved movie night for my family!

### DISCUSSION THREAD

**Discussion by myrison** [LukeChannings/moviematch#94] [General]
@LukeChannings  - thank you for creating this software.  I can't remember the last time at our house that we spent less than 30 minutes trying to find a movie we all agreed on.  Even after the 30+ minutes, usually the disagreement ends with us all watching our own thing, until now.  Thank you for creating this for the community. Very much appreciated!

**Comment by joefred** [comment]
@LukeChannings - I know we are several that feel the same thing as @myrison and we are very much looking forward to V2 with more functions.
Keep up the good work
